### PR TITLE
Fix concurrent modification

### PR DESCRIPTION
**Pull request by samuelkyl**
Fixes https://github.com/zwalsh/button/issues/2

### CODE REVIEW

**Review by zwalsh-toast — APPROVED** [review]

File: src/test/kotlin/sh/zachwal/button/presser/PresserManagerTest.kt [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import kotlinx.coroutines.async
+import kotlinx.coroutines.newFixedThreadPoolContext
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.withContext
+import org.junit.jupiter.api.Test
+
+internal class PresserManagerTest {
+
+    @Test
+    fun concurrentOperationsTest() {
```
Comment (suzannebecker-toast) [inline src/test/kotlin/sh/zachwal/button/presser/PresserManagerTest.kt:13]: should there be some assertions in here? or does this just test that no exn is thrown lol

File: src/main/kotlin/sh/zachwal/button/presser/PresserManager.kt [hunk 2]
```diff
@@ -9,11 +9,10 @@ class PresserManager : PresserObserver {
 
     private val currentlyPressing = mutableSetOf<Presser>()
 
-    @Synchronized
     private suspend fun update() {
         val pressingCount = currentlyPressing.count()
         logger.info("Pressing count now $pressingCount")
-        pressers.forEach { presser ->
+        pressers.toTypedArray().forEach { presser ->
```
Comment (suzannebecker-toast) [inline src/main/kotlin/sh/zachwal/button/presser/PresserManager.kt:15]: could you add a comment explaining this fix

**Review by suzannebecker-toast — COMMENTED** [review]
just a few comments

File: src/main/kotlin/sh/zachwal/button/presser/PresserManager.kt (hunk 2 above)
Comment (sleungtoast) [reply src/main/kotlin/sh/zachwal/button/presser/PresserManager.kt:15]: no

File: src/test/kotlin/sh/zachwal/button/presser/PresserManagerTest.kt (hunk 1 above)
Comment (sleungtoast) [reply src/test/kotlin/sh/zachwal/button/presser/PresserManagerTest.kt:13]: no. yes.

**Review by suzannebecker-toast — COMMENTED** [review]
🚀

**Pull request merged**